pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -1433,3 +1494,40 @@ func countLines(r io.Reader) (int, error) {
 		}
 	}
 }
+
+// ServiceAccountsMarshaller implements serialization of Service Accounts
+type ServiceAccountsMarshaller struct {
+	v                    []corev1.ServiceAccount
```

Comment:
Can't this var have a better name? It is kind hard to figure out that `v` is a reference to `[]corev1.ServiceAccount`, IMHO.

Reply:
Actually if you look it is a pattern in this file, and furthermore golang prefers short names, see in here: https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
does `v` stand for `value` ?

Reply:
I mean, fine with the shorter name, but `v` is a kinda weird name to `ServiceAccount` shorthand. But, could be that I'm just bikeshedding. :thinking:

Reply:
I think so Marcell, its probably from value, but I can chance it to for example sa. But as I said it is used this way above.